lib/ayesql.ex

```diff
@@ -356,7 +356,7 @@ defmodule AyeSQL do
   ```
   """
   defmacro defqueries(file) do
-    with {:ok, contents}  <- File.read(file),
+    with {:ok, contents}  <- File.read!(file),
```

Comment:
`File.read!` spec (you can check it in IEx with `h File.read!`) specifies that it will never return an `{:ok, binary()}`, but `binary()` and it will fail with an exception if the path is not found. But if we want it to fail on error, maybe is better to add an `else` clause to this `with` statement.